jenkins-scripts/docker/ign_launch-compilation.bash

```diff
@@ -29,29 +29,29 @@ if ! [[ ${IGN_LAUNCH_MAJOR_VERSION} =~ ^-?[0-9]+$ ]]; then
 fi
 
 if [[ ${IGN_LAUNCH_MAJOR_VERSION} -ge 3 ]]; then
-  export BUILD_IGN_COMMON=true
-  export IGN_COMMON_MAJOR_VERSION=3
-  export IGN_COMMON_BRANCH=ign-common3
-
-  export BUILD_IGN_SENSORS=true
-  export IGN_SENSORS_MAJOR_VERSION=4
-  export IGN_SENSORS_BRANCH=master
+  export BUILD_IGN_GAZEBO=true
+  export IGN_GAZEBO_MAJOR_VERSION=4
+  export IGN_GAZEBO_BRANCH=master
 
   export BUILD_IGN_GUI=true
   export IGN_GUI_MAJOR_VERSION=4
   export IGN_GUI_BRANCH=master
 
+  export BUILD_IGN_MSGS=true
+  export IGN_MSGS_MAJOR_VERSION=6
+  export IGN_MSGS_BRANCH=master
+
   export BUILD_IGN_RENDERING=true
   export IGN_RENDERING_MAJOR_VERSION=4
   export IGN_RENDERING_BRANCH=master
 
-  export BUILD_IGN_GAZEBO=true
-  export IGN_GAZEBO_MAJOR_VERSION=4
-  export IGN_GAZEBO_BRANCH=master
+  export BUILD_IGN_SENSORS=true
+  export IGN_SENSORS_MAJOR_VERSION=4
+  export IGN_SENSORS_BRANCH=master
 fi
 
 export USE_GCC8=true
 
 export GZDEV_PROJECT_NAME="ignition-launch${IGN_LAUNCH_MAJOR_VERSION}"
 
-. ${SCRIPT_DIR}/lib/generic-building-base.bash
+e ${SCRIPT_DIR}/lib/generic-building-base.bash
```

Comment:
Did you mean to change this?

Reply:
aaaaah 173b4a6e374806ca9535a0d902d0245f6e5b6da5